src/CommunityToolkit.Maui.Core/Interfaces/IPopup.shared.cs

```diff
@@ -0,0 +1,61 @@
+﻿using IElement = Microsoft.Maui.IElement;
+using LayoutAlignment = Microsoft.Maui.Primitives.LayoutAlignment;
+
+namespace CommunityToolkit.Maui.Core;
+
+/// <summary>
+/// Represents a small View that pops up at front the Page.
+/// </summary>
+public interface IPopup : IElement
+{
+	/// <summary>
+	/// Gets the View that Popup will be anchored.
+	/// </summary>
+	IView? Anchor { get; }
+	
+	/// <summary>
+	/// Gets the Popup's color.
+	/// </summary>
+	Color? Color { get; }
+
+	/// <summary>
+	/// Gets the Popup's Content.
+	/// </summary>
+	IView? Content { get; }
+
+	/// <summary>
+	/// Gets the horizontal aspect of this element's arrangement in a container.
+	/// </summary>
+	LayoutAlignment HorizontalOptions { get; }
+	
+	/// <summary>
+	/// Gets the LightDimiss property.
+	/// </summary>
+	bool IsLightDismissEnabled { get; }
+	
+	/// <summary>
+	/// Gets the Popup's size.
+	/// </summary>
+	Size Size { get; }
+
+	/// <summary>
+	/// Gets the vertical aspect of this element's arrangement in a container.
+	/// </summary>
+	LayoutAlignment VerticalOptions { get; }
+
+	/// <summary>
+	/// Occurs when the Popup is Closed.
+	/// </summary>
+	/// <param name="result">Return value from the Popup.</param>
+	void OnDismissed(object? result);
+
+	/// <summary>
+	/// Occurs when the Popup is Opened.
+	/// </summary>
+	void OnOpened();
+
+	/// <summary>
+	/// Occurs when the Popup is closed by a LightDismiss.
+	/// </summary>
+	void LightDismiss();
```

Comment:
Could we consider some alternatives such as:

`Close()` - called by close button
`Dismiss()` - popup is dismissed rather than closed

Or with a parameter

`Dismiss(bool wasClosed)`

Reply:
Just adding some research here:

Light Dismiss seems to be a Windows term: https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.primitives.popup.islightdismissenabled?view=winrt-22000

It does not seem to be an Android term: https://www.google.com/search?q=light+dismiss+Android

It does not seem to be an iOS term: https://www.google.com/search?q=light+dismiss+ios

Reply:
On android is the [Cancel](https://developer.android.com/reference/android/app/Dialog#cancel()) method
On iOS is the [ModalPresentation](https://developer.apple.com/documentation/uikit/uiviewcontroller/3229894-modalinpresentation)

So I believe that `Light Dismiss` was used to be the _best_ name (at that time), but I'm open to suggestions

Reply:
Hey Gang! Now is probably a good time to update/resolve this conversation now that we're getting close to merging Popup!

My vote is to move forward with Shaun's naming recommendation:
-  `OnClosed()` 
    - Triggers when the user taps a close button to dismiss a Popup 
- `OnDismissed()` 
    -  Triggers when the user taps outside of the Popup to dismiss it

As someone who primarily works on iOS + Android (and rarely works on Windows), I was unfamiliar with Windows' `Light Dismiss` nomenclature and I personally had to google `LightDismiss` in order to  understand this feature.

My concern is that other devs may have a similar experience and using `Light Dismiss` will also slow down their development velocity and decrease their discoverability of this feature.

That being said, I'm happy to be outvoted if most folks **are** familiar with the term `Light Dismiss`!